src/components/name-tag.js

```diff
@@ -267,6 +297,7 @@ AFRAME.registerComponent("name-tag", {
     this.isAvatarReady = true;
 
     this.updateDisplayName();
+    this.updatePronouns();
```

Comment:
I think we would need to also account for the pronouns text element when calculating the max width here:
`this.size.x = Math.max(this.size.x, NAMETAG_MIN_WIDTH);`